src/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+</head>
+<body>
+  <section id="page-container">
+    <section id="navbar-container">
+      <nav id="navbar">
+          <div id="logo"></div>
+          <ul id="navbar-links">
+            <li>Pokémons</li>
+            <li>Info</li>
+            <li>Contact</li>
+          </ul>
+      </nav>
+    </section>

```

Comment:
- [ ] Please know that this should be a `header` and not a `section`, and the `page-container` should be a `main` and not a `section` as well, the structure should be `header` then `main` then `footer`, kindly fix this. 😉
- [ ] Please know that as per the requirements, the website should contain a Home page header and navigation similar to the given [mockup](https://github.com/microverseinc/curriculum-javascript/blob/main/group-capstone/js_capstone.md), currently the navigation isn't available it's only text and not a navigation menu, also it should have the pokémon counter on the navigation menu, kindly fix that as well. 😁

  This is how your navigation looks:

  ![image](https://user-images.githubusercontent.com/20567503/198763606-2ec439d9-7eb7-4e4d-a2a4-5118ae3234a1.png)

  And this is the mockup's navigation:

  ![image](https://user-images.githubusercontent.com/20567503/198763747-4dec8a5c-6524-4b4b-ac99-6c267e94d42b.png)

Reply:
Thank you for the keen observations! Changes applied!